src/internal/starlark/lib/k8s/k8s_test.go

```diff
@@ -0,0 +1,80 @@
+package k8s
+
+import (
+	"testing"
+
+	ourstar "github.com/pachyderm/pachyderm/v2/src/internal/starlark"
```

Comment:
If a package will always (even almost always) get imported with a different name, I think it makes sense to rename it.

Reply:
There is no tool that enforces this; goimports will keep doing it if it's already done, but there is no way to mandate that for new packages.  So I think it will be annoying if we make it a rule and there is no way to check if you did it right or not.